Rules/Playbooks/Smugglers.md

```diff
@@ -0,0 +1,105 @@
+
+**You gain XP when...**
+- You meet one of the [[16 Advancement#Crew Advancement|generic advancement criteria]]
+- You **\<add a crew specific advancement trigger>**
+## Exposure
+
+| heat        | wanted level |
+| ----------- | ------------ |
+| `clock 0/9` | `clock`      |
+See the [[03 Heat|heat]] rules for how you manage heat and the consequences it can have.
+## Preferred Cargo
+
+> Choose a preferred cargo from this list and delete the rest.
+- weapons
+- people
+- contraband
+- alien technology
+
+## Contacts
+_All of these are people your crew has connections with. Flesh out that connection as the game goes on._
+- [[Geralt]], a bartender
+- [[Nova]], a shuttle pilot
+- [[Seryn]], a drug dealer
+- [[Roland]], an arms dealer
+- [[Rat]], a scrapper
+- [[Indomitable]], an angel
+
+## Special Abilities
+> Select one of these as part of crew creation. You can get the rest by filling your crew XP track. If you're not sure where to start, pick the first option.
+
+- [ ]  **Like Part of the Family**: Create one of your shuttles as a **cohort** (use the shuttle edges and flaws, below). Its quality is equal to your Tier +1.
+	- Edges
+		- **Nimble**: The vehicle handles easily. Consider this an **assist** for tricky maneuvers. 
+		- **Simple**: The vehicle is easy to repair. Remove all of its harm during downtime. 
+		- **Sturdy**: The vehicle keeps operating even when _broken_.
+	- Flaws
+		- **Costly**: The vehicle costs 1 coin per downtime to keep in operation. 
+		- **Distinct**: The vehicle has memorable features. Take +1 heat when you use it on a score. 
+		- **Finicky**: The vehicle has quirks that only one person understands. When operated without them, them, the vehicle has -1 quality.
+- [ ] **All hands**: During **downtime**, one of your **cohorts** may perform a downtime activity for the crew to **acquire an asset**, **reduce** **heat**, or work on a **long-term project**.
+- [ ] **Alien Acquaintances**: Crew members remain in control of their actions when possessed. (_how did you learn to do this?_)
+- [ ] **Just passing through**: During **downtime**, take **-1 heat**. When your **heat** is 4 or less, you get **+1d** to deceive people when you pass yourselves off as ordinary citizens.
+- [ ] **No traces**: When you keep an operation quiet or make it look like an accident, you get half the **rep** value of the target (round up) instead of zero. When you end downtime with zero **heat** , take **+1** **rep**.
+- [ ] **Ramming speed**: Your vehicle gains armor, and +1 **effect** for damage and speed.
+- [ ] **Pack rats**: Your lair is a jumble of stolen items. When you roll to **acquire an asset**, take **+1d**.
+- [ ] <input type="checkbox"/>**Veteran**: Pick a special ability from another playbook.
+
+
+
+## Upgrades
+### Lair
+- [ ] <input type="checkbox"/> vault
+- [ ] quarters
+- [ ] <input type="checkbox"/> secure
+- [ ] workshop
+- [x] <input type="checkbox"/> shuttle
+- [ ] hidden
+- [ ] <input type="checkbox"/> computer
```

Comment:
We've chatted about this a few times but theres still something immensely entertaining about any old computer as an upgrade. At the risk of overcomplicating, I wonder if we'd want to use tags with computers like _fine_ theives tools does; _rudimentary_ computer, _powerful_ computer, _connected_ computer?

Reply:
Yeah, I can imagine that making sense; tags always make things more interesting. Untagged, I'd imagine this represents something disconnected and not particularly powerful. A modern laptop whose wifi is broken 😆